concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -1,197 +1,362 @@
 #!/usr/bin/env python3
-
+import argparse
 import os
 import sys
 import datetime
 import hashlib
-import time
 import random
-from base64                 import b64encode
+import requests
+import time
 
-from golem_messages         import message
-from golem_messages         import shortcuts
+from base64                          import b64encode
+from golem_messages                  import message
+from golem_messages                  import shortcuts
+from golem_messages.message.concents import AckForceGetTaskResult
+from golem_messages.message.concents import ForceGetTaskResultFailed
+from golem_messages.message.concents import ForceGetTaskResultUpload
 
-from utils.helpers          import get_current_utc_timestamp
-from utils.testing_helpers  import generate_ecc_key_pair
+from utils.helpers                   import get_current_utc_timestamp
+from utils.testing_helpers           import generate_ecc_key_pair
 
-from api_testing_helpers    import api_request
-from api_testing_helpers    import timestamp_to_isoformat
-
-from freezegun              import freeze_time
-import requests
+from api_testing_common              import api_request
+from api_testing_common              import create_task_to_compute
+from api_testing_common              import get_protocol_constants
+from api_testing_common              import print_protocol_constants
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concent_api.settings")
 
-(PROVIDER_PRIVATE_KEY,  PROVIDER_PUBLIC_KEY)  = generate_ecc_key_pair()
-(REQUESTOR_PRIVATE_KEY, REQUESTOR_PUBLIC_KEY) = generate_ecc_key_pair()
-
+DEADLINE_OFFSET        = 5
```

Comment:
What is `DEADLINE_OFFSET`? This needs an explanatory comment because it's not obvious.

If I had to guess, I'd say it's the difference between `TaskToCompute.deadline` and `TaskToCompute.timestamp`. Is that right? If that's the case then I think it needs a better name. For example `DEFAULT_TASK_DURATION`, though maybe you can come up with a better one. Something that would suggest that (1) it's not really a protocol constant but just a default you're using here (tasks do not have constant durations; it completely depends on what requestor and provider agree) and (2) that it's the task duration, not some offset that you add on top of the task deadline.

Reply:
agreed; changed to DEFAULT_TASK_DURATION